docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/clus_maint.md

```diff
@@ -1,30 +1,32 @@
 
 !!! note
-    If you are not using horizontal scaling in your environment, any fix that requires a restart results in temporary outage for your users. Existing 24x7 installation procedures do not apply to these environments.
+    If you don't use horizontal scaling in your environment, any fix that requires restarting equipment results in a temporary outage for your users. Existing 24x7 installation procedures do not apply to these environments.
 
-!!! warning
-    You do not need to remove the node from the cluster when you apply fixes; doing so might result in the inability to add the node back to the cluster.
+!!! important
+    You do not have to remove the node from the cluster when you apply fixes. If you do remove a node, you might be unable to add the node back to the cluster.
 
 ## Minor fixes
 
-Apply all minor fixes on each cluster node. Use the installation instructions that are supplied with the fix. When you apply minor fixes that might update previously deployed enterprise applications, first turn off the auto-synchronization feature of the deployment manager. After the fix is deployed on all cluster nodes, you can force a manual synchronization with the deployment manager to ensure that all updates are synchronized on the nodes. You can then enable the auto-synchronization feature again.
+Apply all minor fixes on each cluster node. Use the installation instructions that come with the fix. When you apply minor fixes that might update previously deployed enterprise applications<!-- How would users determine that a fix might update in this manner? -->, first turn off the auto-synchronization feature of the deployment manager. After you deploy the fix on all cluster nodes, you can force a manual synchronization with the deployment manager to ensure that all updates are synchronized on the nodes. Then, you can enable the auto-synchronization feature again.
 
-If the documentation associated with the minor fix requires that you restart HCL Portal or WebSphere Application Server, apply the minor fix one node at a time. This process enables other nodes to continue to provide service to your users. However, if the fix requires an update to the HCL Portal databases, you might be required to stop the cluster before you apply the fix. If so, use a procedure that ensures 24x7 availability.
+If the documentation associated with the minor fix requires that you restart HCL Portal or WebSphere Application Server, apply the minor fix one node at a time. This process enables other nodes to continue to provide service to users. However, if the fix requires an update to the HCL Portal databases, you might be required to stop the cluster before you apply the fix. If so, use a procedure that ensures 24x7 availability.
```

Comment:
If the documentation associated with the minor fix requires that you restart HCL DX or WebSphere Application Server, apply the minor fix one node at a time. This process enables other nodes to continue to provide service to the users. However, if the fix requires an update to the HCL DX databases, you might be required to stop the cluster before you apply the fix. In that case, use a procedure that ensures 24x7 availability.